net.krafting.PedantiK.yml

```diff
@@ -0,0 +1,82 @@
+app-id: net.krafting.PedantiK
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: PedantiK.py
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: OpenBLAS
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - -DBUILD_SHARED_LIBS=ON
+    - -DBUILD_TESTING=OFF
+    sources:
+      - type: git
+        url: https://github.com/OpenMathLib/OpenBLAS
+        tag: 'v0.3.26'
+        commit: 6c77e5e314474773a7749357b153caba4ec3817d
+
+  - name: LAPACK
```

Comment:
does it really need lapack and blas for that?

Reply:
I had issue when one wasn't present I believe, I'll try again on my side.

Reply:
I mean why do you need computational libraries for a game? (includes numpy).

Reply:
gensim requires them both iirc (maybe I'm wrong, I'm re-testing all of that)

I use it to find words close to other words and such

Reply:
ah ok.

Reply:
Indeed, it does not seems to be required. It build just fine without it.